Engine/Component/ComponentAudioSource.h

```diff
@@ -41,14 +41,16 @@ class ComponentAudioSource : public Component
 
 	void SetListener(const AkGameObjectID listener_name);
 
-	std::string awake_event;
+
+public:
+	std::string selected_event;
```

Comment:
Store it as an index to the events array, it's more efficient and this way we ensure that the selected event exists.